contentcuration/automation/utils/appnexus/base.py

```diff
@@ -1,46 +1,183 @@
+import time
+import logging
+import requests
 from abc import ABC
 from abc import abstractmethod
 from builtins import NotImplementedError
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+from . import errors
+
+
+class SessionWithMaxConnectionAge(requests.Session):
+    """
+        Session with a maximum connection age. If the connection is older than the specified age, it will be closed and a new one will be created.
+        The age is specified in seconds.
+    """
+    def __init__(self, age = 10):
```

Comment:
Could we increase this to `100` by default?

Reply:
Done =)